### DISCUSSION TITLE
How to setup a second instace of Firefly

### DISCUSSION THREAD

**Discussion by Londoneye02** [firefly-iii/firefly-iii#9311] [Firefly III] [answered]
Hello. I have my firefly installation in docker, (using docker compose) and I would like to have a duplicated instace to make some tests with the API and leave my original one untouched.

For duplicating the firefly and even the importer I think I got it covered, but my doubt is about the database. Wich settings do I need to change so that i don't mess up with the original one and the firefly installation points to the copied database and not the original one?

**Comment by JC5** [comment, marked as answer]
If you use docker compose, putting the docker compose file (together with the env file) in a new directory is enough to create a new set.

**Reply by Londoneye02** [reply]
Thank you for  super fast answer!!!!

So just changing the ports on the new firefly instace is enough?

**Reply by JC5** [reply]
That should do it, yes!
